third_party/nvfuser/csrc/utils.cpp

```diff
@@ -149,7 +149,8 @@ auto parseDisableOptions() {
       {"index_hoist", DisableOption::IndexHoist},
       {"nvtx", DisableOption::Nvtx},
       {"predicate_elimination", DisableOption::PredicateElimination},
-      {"welford_vectorization", DisableOption::WelfordVectorization}};
+      {"welford_vectorization", DisableOption::WelfordVectorization},
+      {"nvfuser_zero", DisableOption::MagicZero}};
```

Comment:
nit: Let's make the option string be "magic_zero".

Reply:
changed